packages/dds/sequence/src/sequence.ts

```diff
@@ -86,7 +82,7 @@ export abstract class SharedSegmentSequence<T extends MergeTree.ISegment>
                 case MergeTree.MergeTreeDeltaType.INSERT:
                     ops.push(MergeTree.createInsertOp(
                         r.position,
-                        cloneDeep(r.segment.toJSONObject())));
+                        r.segment.clone().toJSONObject()));
```

Comment:
Add UT to hitting the bug? Relying on todo test is Ok, but UT would be even better!

Reply:
agreed. so far i can't hit the issue in ut's.